packages/es-components-via-theme/index.js

```diff
@@ -416,13 +415,15 @@ const theme = {
           paddingBottom: '4px',
           textTransform: 'uppercase'
         }
-      },
+      }
     }
   },
-  validationIconName: {
-    success: 'ok',
+  iconNames: {
+    success: 'ok-circle',
+    info: 'info-circle',
     warning: 'exclamation-sign',
-    danger: 'remove'
```

Comment:
On one hand I think this is perfectly fine. On the other hand this will cause snapshots downstream to break, as `validationIconName` gets baked into the generated snapshot files.